scripts/benchmark.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import json
+import re
+
+def load_json(filename):
+    with open(filename) as f:
+        data = json.loads(f.read())
+    return data
```

Comment:
What is the advantage of requiring `json`? IMHO it's easier to just use the plain format since this is created by the benchmark by default.

Reply:
I checked that it works but I don't necessarily want to create the output on hte same nodes that I am running and copying the much shorter plain format it just easier.

Reply:
For me, the advantage of using JSON is that we already have to dump it if we want to use `compare_bench.py` script for run comparison. But you bring a valid point that removing the old "one-line" console output may be stupid. I feel that it brings few points worth discussing:
- With JSON, I believe that we could figure out a way to parse each file exactly once, as we are able to store file as a dictionary in a variable. I think this is challenging with console output
- The rates output is done in human form, using `k` or `M` suffix to indicate thousands/millions. This leads to extra complexity.

This is not to say that I'm opposed to bringing parsing output back. It does have its benefit in its simplicity.

Reply:
@masterleinad I added the console output file reader.